source/whats-new.txt

```diff
@@ -74,6 +74,19 @@ What's New in 4.11
 
 .. New features of the 4.11 driver release include:
 
+- The following network address-related methods are deprecated and will be removed
+  in v5.0:
+
+  - The `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__
+    methods ``getSocketAddress()`` and ``getSocketAddresses()``.
+    Instead of ``getSocketAddress()``, use the ``getAddress()`` method from the
+    ``InetSocketAddress`` class. Instead of ``getSocketAddresses()``, use the
```

Comment:
"In the code at [ServerAddress.java#L191](https://github.com/mongodb/mongo-java-driver/blob/5455ee996f06fee1b5db22b02dc9ab0bd4216913/driver-core/src/main/com/mongodb/ServerAddress.java#L191C41-L191C41), it is recommended to use the `getByName(String)` method from `InetAddress` instead of` getSocketAddress()`. Could you please clarify the source of the information suggesting the use of` getSocketAddress()` from `InetSocketAddress`?

Reply:
The same for [getUnixSocketAddress()](https://github.com/mongodb/mongo-java-driver/blob/481e793011f8db70efc3b8042bb93860a5dfb686/driver-core/src/main/com/mongodb/UnixServerAddress.java#L56). In the code suggested replacement is different from the doc.

Reply:
Thank you for these corrections -- I was just looking at the API docs and clearly made some false assumptions! Let me know if this section looks good to go now.